packages/integration/src/gitlab/core.test.ts

```diff
@@ -49,23 +49,23 @@ describe('gitlab core', () => {
       {
         config: configWithNoToken,
         url:
-          'https://gitlab.com/groupA/teams/teamA/subgroupA/repoA/-/blob/branch/my/path/to/file.yaml',
+          'https://gitlab.com/groupA/teams/teamA/subgroupA/repoA/-/blob/branch/my/path%20with%20spaces/to/file.yaml',
```

Comment:
@adrianke77  `https://gitlab.com/groupA/teams/teamA/subgroupA/repoA/-/blob/branch/my/path/to/file.yaml` should also work right?

IMO, we should add another test where we show that both these file path types work. :)